pandas/core/base.py

```diff
@@ -745,7 +745,7 @@ def nbytes(self):
         """
         Return the number of bytes in the underlying data.
         """
-        return self._values.nbytes
```

Comment:
I think this is the wrong property that is being changed? (`nbytes`, not `size`)

Reply:
Whoops, will fix that soon.